opentelemetry-instrumentation/src/opentelemetry/instrumentation/auto_instrumentation/components.py

```diff
@@ -110,10 +120,30 @@ def import_exporters(
     return trace_exporters, metric_exporters
 
 
+def import_ids_generator(ids_generator_name: str) -> trace.IdsGenerator:
+    ids_generator_impl = None
+
+    for id_generator in iter_entry_points("opentelemetry_ids_generator"):
+        if id_generator.name == ids_generator_name.strip():
```

Comment:
why deviate from the behavior of the exporters and perform a strip on the passed generator name?

Although it helps in a rare handful of cases, I think it's better to just perform no mutation on user input. 

Alternatively, exporter and ids_generator should have the same mutation: strip both or strip none. It's minor, but the most confusing thing to a user is inconsistency.

Reply:
I was actually under the impression that exporter _did_ strip the value:

https://github.com/open-telemetry/opentelemetry-python/blob/8660deaa22fb08674229a95ffb858cfa48d5bfa7/opentelemetry-instrumentation/src/opentelemetry/instrumentation/auto_instrumentation/components.py#L48-L49

Either way I think stripping the value is a good idea, I would expect

```
opentelemetry-instrument --ids-generator aws_xray
```

and

```
opentelemetry-instrument --ids-generator    aws_xray       
```

to work the same. But not sure if my second example would have whitespace to begin with :sweat_smile: